thirdparty/lunajson/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Build in source tree.
+set(BINARY_DIR ${SOURCE_DIR})
+
+append_install_commands(INSTALL_CMD src/lunajson.lua DESTINATION common)
+append_tree_install_commands(INSTALL_CMD src/lunajson common/lunajson)
+
+ko_write_gitclone_script(
+    GIT_CLONE_SCRIPT_FILENAME
+    https://github.com/grafi-tt/lunajson
+    3d10600874527d71519b33ecbb314eb93ccd1df6
+    ${SOURCE_DIR}
+)
+
+external_project(
+    DOWNLOAD_COMMAND ${CMAKE_COMMAND} -P ${GIT_CLONE_SCRIPT_FILENAME}
+    INSTALL_COMMAND ${INSTALL_CMD}
+)
```

Comment:
There does not seem to be any advantage in using the latest commit over the latest tag 1.2.3: https://github.com/grafi-tt/lunajson/compare/1.2.3...master.


```suggestion
external_project(
    URL https://github.com/grafi-tt/lunajson/archive/refs/tags/1.2.3.tar.gz
    URL_MD5 50236a29ac4222ce6a83dc4de757666a
    INSTALL_COMMAND ${INSTALL_CMD}
)
```